zerver/migrations/0469_add_topic_wildcard_mentioned_flag_and_create_index.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.2.3 on 2023-08-16 14:16
+
+import bitfield.models
+from django.contrib.postgres.operations import AddIndexConcurrently, RemoveIndexConcurrently
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("zerver", "0468_rename_followup_day_email_templates"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="archivedusermessage",
+            name="flags",
+            field=bitfield.models.BitField(
+                [
+                    "read",
+                    "starred",
+                    "collapsed",
+                    "mentioned",
+                    "wildcard_mentioned",
+                    "summarize_in_home",
+                    "summarize_in_stream",
+                    "force_expand",
+                    "force_collapse",
+                    "has_alert_word",
+                    "historical",
+                    "is_private",
+                    "active_mobile_push_notification",
+                    "topic_wildcard_mentioned",
+                ],
+                default=0,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="usermessage",
+            name="flags",
+            field=bitfield.models.BitField(
+                [
+                    "read",
+                    "starred",
+                    "collapsed",
+                    "mentioned",
+                    "wildcard_mentioned",
+                    "summarize_in_home",
+                    "summarize_in_stream",
+                    "force_expand",
+                    "force_collapse",
+                    "has_alert_word",
+                    "historical",
+                    "is_private",
+                    "active_mobile_push_notification",
+                    "topic_wildcard_mentioned",
+                ],
+                default=0,
+            ),
+        ),
+        AddIndexConcurrently(
+            model_name="usermessage",
+            index=models.Index(
+                "user_profile",
+                "message",
+                condition=models.Q(
+                    ("flags__andnz", 8),
+                    ("flags__andnz", 16),
+                    ("flags__andnz", 8192),
+                    _connector="OR",
+                ),
+                name="zerver_usermessage_any_mentioned_message_id",
+            ),
+        ),
```

Comment:
This step is the main expensive part of the migration, and it's important that this migration be applied before we restart Django into the new code, as the `is:mentioned` narrow will do a table scan without it.

I believe it can be pre-applied though, the new column is always 0 since we don't have any topic mentions yet; @alexmv FYI.